recipes/chaco/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "chaco" %}
+{% set version = "5.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chaco-{{ version }}.tar.gz
+  sha256: c29308b243b683a439b2345d043a9c2be07aae034582bfd1a2c037d35716008d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython
+    - oldest-supported-numpy
```

```suggestion
    - numpy
```
We don't have an oldest-numpy package.